analysis/descriptives/table2.R

```diff
@@ -127,4 +128,4 @@ table2[,setdiff(colnames(table2),c("name","cohort","exposure","outcome","analysi
 # Save Table 2 -----------------------------------------------------------------
 print('Save rounded Table 2')
 
-write.csv(table2, paste0("output/table2_",cohort,"_rounded.csv"), row.names = FALSE)
\ No newline at end of file
+write.csv(table2, paste0("output/table2_",cohort,"_midpoint6.csv"), row.names = FALSE)
```

Comment:
If you haven't run Table 2 yet, you can fix the midpoint6 rounding here at the source and use `make_other_output.R` to create the Table 2 output. The file `make_table2_output.R` is a hack if you have already run Table 2 (which takes a long time to run) but need to fix the midpoint6 rounding for outputting. This was discussed on Slack [here](https://grp-ehr.slack.com/archives/C03RWTCBVT9/p1699019578661349).

Reply:
Changed script.

Reply:
Point 1 is not clear. Point 2 done. Point 3 cannot be done due to not clear instructions.